p2p/node.go

```diff
@@ -361,12 +370,54 @@ func (n *QriNode) SendMessage(ctx context.Context, msg Message, replies chan Mes
 	return nil
 }
 
+// connected is called when a connection opened via the network notifee bundle
+func (n *QriNode) connected(_ net.Network, conn net.Conn) {
+	log.Debugf("connected to peer: %s", conn.RemotePeer())
+	pi := n.Host().Peerstore().PeerInfo(conn.RemotePeer())
+	n.pub.Publish(context.Background(), event.ETP2PPeerConnected, pi)
+}
+
+func (n *QriNode) disconnected(_ net.Network, conn net.Conn) {
+	log.Debugf("disconnected from peer: %s", conn.RemotePeer())
+	pi := n.Host().Peerstore().PeerInfo(conn.RemotePeer())
+	n.pub.Publish(context.Background(), event.ETP2PPeerDisconnected, pi)
+}
+
 // QriStreamHandler is the handler we register with the multistream muxer
 func (n *QriNode) QriStreamHandler(s net.Stream) {
 	// defer s.Close()
 	n.handleStream(WrapStream(s), nil)
 }
 
+func (n *QriNode) libp2pSubscribe() error {
+	host := n.host
+	sub, err := host.EventBus().Subscribe([]interface{}{
+		new(libp2pevent.EvtPeerIdentificationCompleted),
+		new(libp2pevent.EvtPeerIdentificationFailed),
+	},
+	// libp2peventbus.BufSize(1024),
```

Comment:
did you research the purpose of this value? Any reason for keeping this commented out?